lib/db/models/Node.ts

```diff
@@ -34,6 +34,7 @@ export default (sequelize: Sequelize.Sequelize, DataTypes: Sequelize.DataTypes)
       },
     },
     banned: { type: DataTypes.BOOLEAN, allowNull: true },
+    inbound: { type: DataTypes.BOOLEAN, allowNull: true },
```

Comment:
I don't think it makes sense to store this in the database. A node isn't permanently inbound nor permanently outbound, at least as I see it. We shouldn't want to permanently make the connection one-way only. What if a peer connects to us, but then we lose the ability to receive incoming connections? We'd still want to be able to make an outgoing connection to that node.

Not to mention, I believe this would be a breaking change that would require a migration script or deleting existing databases, so we should avoid if possible.

Reply:
Thanks for the review @sangaman - I think what you said makes a lot sense. I'll look into the reconnection/peer discovery logic today to see if we can develop a solution without a breaking change.